pyproject.toml

```diff
@@ -65,6 +65,10 @@ dev = [
     "uv",
     "pre-commit",
 ]
+nomad = [
```

Comment:
This is what I needed when developing in an empty venv were first nomad was installed and then pynxtools was installed using -e ".[dev,docs]" as otherwise the here mentioned libraries would have gotten deinstalled and replaced by other versions of these respectively causing startup issues in nomad

Reply:
I do not like this either, would have even spotted if I would have just installed without dev,docs that there are such side effects.